requirements/base.txt

```diff
@@ -3,4 +3,5 @@ pandas>=0.23,<=0.24
 google-cloud-storage>=1.16,<=1.18
 mock>=3.0.5
 pyarrow>=0.14
-pydata-google-auth>=0.1.3
\ No newline at end of file
+pydata-google-auth>=0.1.3
+apache-airflow>=1.10.6
```

Comment:
We don't want airflow as a base requirements, add it as an additional requirement or don't add it at all(it is used only when deployed on airflow or when unit testing our code).